data_structures/queue_implementation.cpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * @author [saqlain] (https://github.com/Mysterious786)
+ * @file
+ *
+ * Implementation of a Queue using Single Stacks.
+ *
+ */
+
+#include <iostream>  /// for IO operations
+#include <stack>     /// for stack operations
+#include <cstdlib>   /// for datatype conversion
+namespace
+{
+    /**
+     * Queue data structure.Store elements in FIFO
+     * (first-in-first-out) manner.
+     *
+     */
+
+    class Queue
+    {
+        std::stack<int> s;
+
+    public:
+        /**
+         * Pushes data to the back of queue
+         */
+        void enqueue(int data)
```

Comment:
Parameters should be documented as well.

Reply:
Can we rename the argument to item than data ?